hack/build-image

```diff
@@ -724,8 +1044,31 @@ def main():
         action="store_const",
         dest="main_action",
         const=retag,
-        help=("Regenerate any short (unqualified) tags expected to exist"
-              " for a given FQIN. Requires FQIN to already exist locally."),
+        help=(
+            "Regenerate any short (unqualified) tags expected to exist"
+            " for a given FQIN. Requires FQIN to already exist locally."
+        ),
+    )
+    behaviors.add_argument(
+        "--rebuild",
+        action="store_const",
+        dest="main_action",
+        const=rebuild,
+        help="Always rebuild images even if matching images exist locally.",
+    )
```

Comment:
How is this different from `--image-condition=rebuild`? In other words can we live with either one of them?

Reply:
Image condition only affects commands like push or archive that expect an image and allow it to rebuild or not. --rebuild is a full blown top-level command.

If I had a time machine I'd do the cli different, but I don't want to completely redo it now. Especially since that will break the existing consumers and my own workflows.

Reply:
Ah..I get it now, thanks.